README.md

```diff
@@ -65,15 +73,128 @@ href="https://github.com/GoogleCloudPlatform/terraform-example-foundation-app/tr
+
+- **3 GKE Clusters**
+  - Cluster1 in the primary region (us-east1)
+  - Cluster2 in the secondary region (us-west1)
+  - MCI Cluster in the primary region (us-east1)
+- **Bastion Host VM** in the secondary region (us-west1)
+- **2 Postgres CLoudSQL instances** in the primary and secondary regions, respectively
+- **Secret** to store the CloudSQL Admin Password
+- **4 KMS Keyrings and Keys**
+  - 2 KMS Keyrings and Keys for GKE, one in each region
+  - 2 KMS Keyrings and Keys for CloudSQL, one in each region
+- **Service Account for KMS** to own/manage the Keyrings and Keys
+- **Service Account for the Bastion Host VM** with roles to install Anthos Service Mesh
+- **4 Log Sinks**, one in each project
+- **Log Sink Destination Storage Bucket** that Log Sinks write logs to
+- **Cloud Armor Policy**
+- **External IP** for accessing the application externally
+
+### [6. anthos-install](./6-anthos-install)
+
+The purpose of this step is to install the Anthos components required for the Bank of Anthos example application - Anthos Config Management and Anthos Service Mesh. **This step in the process is currently manual.**  Anthos Config Management can help you create a common configuration across all your infrastructure, including custom policies, and apply it both on-premises and across clouds. Anthos Service Mesh lets you easily manage the security and telemetry of complex environments.  Currently, this includes:
+
+- All required components to install **Anthos Config Management**
+- All required components to install **Anthos Service Mesh**
+
+### [7. build_app](./build_app)
+
+The purpose of this step is to utilize an opinionated repository to demonstrate Cloud Build based builds of Bank of Anthos with secure CI/CD principles applied. The example herein simulates a company building and deploying the Bank of Anthos example application to a multi-tier kubernetes cluster using asynchronous GitOps. Currently, this includes:
+
+1. **Source Code**
+   - Unit tests are run for all source code
+   - Static code analysis is performed on all source code
+   - Secrets scanner looks for secrets embedded in source code
+   - Code coverage numbers are pulled to make a decision based on the results
+
+1. **Artifact Verification**
+   - Container structure tests to verify that the container built in the previous step conforms to organizational standards
+   - Container analysis to verify that the container does not contain Common Vulnerabilities and Exposures
+
+1. **Security Attestation**
+   - Creates an an attestation for the artifact
+
 ## Source Code Headers
```

Comment:
Lets remove this part

Reply:
removed in 5ab064bf9e45ea280a7f72d103b9aec6e3fdfae2